apps/crn-server/test/fixtures/research-output.fixtures.ts

```diff
@@ -341,14 +341,16 @@ export const getResearchOutputUpdateData = (): ResearchOutputUpdateData => ({
   updatedBy: 'userId',
 });
 
-export const getRestResearchOutputUpdateData =
-  (): InputResearchOutput['data'] => {
-    const { createdBy, ...restResearchOutputData } =
-      getRestResearchOutputCreateData();
-    return {
-      doi: { iv: null },
-      accession: { iv: null },
-      rrid: { iv: null },
-      ...restResearchOutputData,
-    };
+export const getRestResearchOutputUpdateData = (): Omit<
+  InputResearchOutput['data'],
+  'addedDate'
```

Comment:
`InputResearchOutput` is a type that we use for squidex payload and i think that the addedDate should be present in the fixtures still?